templates/repo/issue/view_content/pull.tmpl

```diff
@@ -317,112 +323,105 @@
-									const defaultMergeMessage = 'Reviewed-on: ' + {{$.Issue.HTMLURL}} + '\n' + {{$approvers}};
-									const mergeForm = {
-										'baseLink': {{.Link}},
-										'textCancel': {{$.i18n.Tr "cancel"}},
-										'textDeleteBranch': {{$.i18n.Tr "repo.branch.delete" .HeadTarget}},
+									'canMergeNow': {{$canMergeNow}},
+									'allOverridableChecksOk': {{not $notAllOverridableChecksOk}},
+									'pullHeadCommitID': {{.PullHeadCommitID}},
+									'isPullBranchDeletable': {{.IsPullBranchDeletable}},
+									'defaultDeleteBranchAfterMerge': {{$prUnit.PullRequestsConfig.DefaultDeleteBranchAfterMerge}},
+									'mergeMessageFieldPlaceHolder': {{$.i18n.Tr "repo.editor.commit_message_desc"}},
 
-										'allOverridableChecksOk': {{not $notAllOverridableChecksOk}},
-										'pullHeadCommitID': {{.PullHeadCommitID}},
-										'isPullBranchDeletable': {{.IsPullBranchDeletable}},
-										'defaultDeleteBranchAfterMerge': {{$prUnit.PullRequestsConfig.DefaultDeleteBranchAfterMerge}},
-										'mergeMessageFieldPlaceHolder': {{$.i18n.Tr "repo.editor.commit_message_desc"}},
-									};
-									mergeForm['mergeStyles'] = [
-										{
-											'name': 'merge',
-											'allowed': {{$prUnit.PullRequestsConfig.AllowMerge}},
-											'textDoMerge': {{$.i18n.Tr "repo.pulls.merge_pull_request"}},
-											'mergeTitleFieldText': defaultMergeTitle,
-											'mergeMessageFieldText': defaultMergeMessage,
-										},
-										{
-											'name': 'rebase',
-											'allowed': {{$prUnit.PullRequestsConfig.AllowRebase}},
-											'textDoMerge': {{$.i18n.Tr "repo.pulls.rebase_merge_pull_request"}},
-											'hideMergeMessageTexts': true,
-										},
-										{
-											'name': 'rebase-merge',
-											'allowed': {{$prUnit.PullRequestsConfig.AllowRebaseMerge}},
-											'textDoMerge': {{$.i18n.Tr "repo.pulls.rebase_merge_commit_pull_request"}},
-											'mergeTitleFieldText': defaultMergeTitle,
-											'mergeMessageFieldText': defaultMergeMessage,
-										},
-										{
-											'name': 'squash',
-											'allowed': {{$prUnit.PullRequestsConfig.AllowSquash}},
-											'textDoMerge': {{$.i18n.Tr "repo.pulls.squash_merge_pull_request"}},
-											'mergeTitleFieldText': defaultSquashMergeTitle,
-											'mergeMessageFieldText': defaultMergeMessage,
-										},
-										{
-											'name': 'manually-merged',
-											'allowed': {{and $prUnit.PullRequestsConfig.AllowManualMerge $.IsRepoAdmin}},
-											'textDoMerge': {{$.i18n.Tr "repo.pulls.merge_manually"}},
-											'hideMergeMessageTexts': true,
-										}
-									];
-									window.config.pageData.pullRequestMergeForm = mergeForm;
-								})();
-							</script>
+									'hasPendingPullRequestMerge': {{.HasPendingPullRequestMerge}},
+									'hasPendingPullRequestMergeTip': {{$hasPendingPullRequestMergeTip}},
+								};
+								mergeForm['mergeStyles'] = [
```

Comment:
'hideAutoMerge': true ... should be true if you can merge directly ...

Reply:
Updated.